src/components/Dashboard/Header/index.tsx

```diff
@@ -33,7 +35,7 @@ const DashboardHeader = ({
         {shouldShowShimmer ? (
           <Shimmer classAttributes="w-[117px] h-[18px] rounded-[14px]" />
         ) : (
-          <p className="text-x-small font-extrabold">{dateTime}</p>
+          <p className="text-x-small font-extrabold">{dateTime.toUpperCase()}</p>
```

Comment:
```suggestion
          <p className="text-x-small font-extrabold uppercase">{dateTime}</p>
```

Reply:
@nvminhtue fixed in [29fe872](https://github.com/manh-t/react-survey/pull/61/commits/29fe8721eeffdb78f076407cab85b9c967a5f030)